### ISSUE TITLE
Add/delete layer in pixel.js has disappeared

### ISSUE THREAD

**Issue by kyrieb-ekat** [DDMAL/Rodan#1151]
<img width="1116" alt="Screenshot 2024-05-23 at 12 42 18 PM" src="https://github.com/DDMAL/Rodan/assets/74431986/6dcb7d89-3fab-4e39-822b-e73c19e96f3d">
The tutorial contains instructions for adding/removing layers as you proceed in your labeling; these buttons are not present. See screenshot. 

Did we remove these earlier, and not update the tutorial, maybe? If so, then the tutorial might need to be tweaked. Otherwise, at the moment there is no way to remove or add layers in pixel.

**Comment by yinanazhou** [comment]
I can reproduce this issue locally in Rodan, but I cannot reproduce this issue in Pixel.js. I tried to run pixel-wrapper locally, but the instructions are outdated. Looking at the code, the buttons should be there.

While debugging this issue, I found that `python3-celery/Dockerfile` cloned `pixel_wrapper:v2.0.1`. However, `v2.0.1` branch doesn't exist. My hypothesis is that Rodan is not using the latest version of pixel. This is probably a devops issue @homework36
